### PR TITLE
Visual regression testing

### PR DESCRIPTION
**Pull request by Elanhant**
This PR adds two new commands:
* `yarn test:a11y` — runs [Axe](https://www.deque.com/axe/) accessibility checks for Storybook stories
* `yarn test:visual-regression` — performs visual regression testing for Storybook stories

`yarn test` will also run visual regression testing but it **will not** run a11y tests since there are about 25 failures already that we need to address before enabling `test:a11y` to our CI 🙂 

Both new commands rely on [StoryShots addon](https://github.com/storybookjs/storybook/tree/master/addons/storyshots/storyshots-core) as well as its [puppeteer addon](https://www.npmjs.com/package/@storybook/addon-storyshots-puppeteer). 

I've also fixed some a11y issues in some of the stories.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -130,17 +133,19 @@
     "eslint-plugin-prettier": "^3.1.2",
     "eslint-plugin-react": "^7.12.4",
     "filestack-react": "^3.1.0",
-    "fs": "^0.0.1-security",
     "formik": "^2.0.8",
+    "fs": "^0.0.1-security",
```
Comment (mfrachet) [inline package.json]: I'm curious what is this?

File: __tests__/AccessiblityStoryshots.test.js [hunk 2]
```diff
@@ -1,8 +1,12 @@
 import initStoryshots from "@storybook/addon-storyshots"
+import { axeTest } from "@storybook/addon-storyshots-puppeteer"
 import { render } from "@testing-library/react"
+import { getDefaultStoryshotsConfig } from "../.storybook/storyshotsConfig"
```
Comment (greglobinski) [inline __tests__/AccessiblityStoryshots.test.js]: There is no a named `getDefaultStoryshotsConfig` export in `.storybook/storyshotsConfig`, you forgot to rename `getDefaultPuppeteerConfig` to a new name

**Comment by greglobinski** [comment]
> I'm just not sure what's going on but running the test (not a11y tests) gives me like 98 failing tests. Is that "normal"?

same here, but on my local 128 snapshots fail

File: __tests__/ImageStoryshots.test.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
+import initStoryshots from "@storybook/addon-storyshots"
+import { imageSnapshot } from '@storybook/addon-storyshots-puppeteer';
+import { getDefaultPuppeteerConfig } from "../.storybook/storyshotsConfig";
+
+const baseMatchOptions = {
+  dumpDiffToConsole: true,
```
Comment (greglobinski) [inline __tests__/ImageStoryshots.test.js]: What's the value in this, do you see 'image' diff presentation in console. I see only tones of cryptic characters 

Screenshot from 2020-02-18 15 03 42@2x

File: package.json (hunk 1 above)
Comment (Elanhant) [reply package.json]: I didn't even notice this! `fs` is used by a script to scaffold icons, but it should actually be [file-system](https://www.npmjs.com/package/file-system) package. I will update this!

**Comment by Elanhant** [comment]
Closing this PR in favour of setting up Chromatic CI integration in #207. 
Accessibility tests are still going to be helpful, I'll extract them to a separate PR.

**Pull request closed without merging**